server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -18,15 +18,17 @@ import (
 	"github.com/runatlantis/atlantis/server/neptune/workflows/plugins"
 	"go.temporal.io/sdk/temporal"
 	"go.temporal.io/sdk/workflow"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/lock"
 )
 
 type queue interface {
 	IsEmpty() bool
 	CanPop() bool
 	Pop() (terraform.DeploymentInfo, error)
-	SetLockForMergedItems(ctx workflow.Context, state LockState)
+	SetLockForMergedItems(ctx workflow.Context, state lock.LockState)
 	GetOrderedMergedItems() []terraform.DeploymentInfo
 	GetQueuedRevisionsSummary() string
+	GetLockState() lock.LockState
```

Comment:
Adding these two functions to the interface is breaking the linter.
Solutions:
1. separate queue package from the revision package so that queue struct can be shared without interfaces? (similar to what I did for the Lock package but this seems more complicated)
2. Keep using interfaces, but split it up into 2 interfaces: one for functions used by deploy worker, one for terraform runner. WIP looking into if this can be done cleanly.

Reply:
Edit: resolved by bumping the interfacebloat max setting in golangci.yml instead